### ISSUE TITLE
`source.strings` link is not right.

### ISSUE THREAD

**Issue by mohataher** [atom/autocomplete-plus#743]
Autocomplete provider for strings file with extension `source.strings` links to `objective-c` package in this [wiki](https://github.com/atom/autocomplete-plus/wiki/Autocomplete-Providers) page.

**Comment by wesinator** [comment]
That is correct - strings file grammar is under https://github.com/atom/language-objective-c/blob/master/grammars/strings%20file.cson
